scripts/signup.js

```diff
@@ -0,0 +1,125 @@
+import { ADMIN_EMAIL } from "./base.js";
+import { isValidEmail, isValidPassword } from "./inputValidation.js";
+import { togglePasswordVisibility } from "./togglePasswordVisibility.js";
+
+document.addEventListener("DOMContentLoaded", function () {
+  const emailInput = document.querySelector("#email");
+  const passwordInput = document.querySelector("#password");
+  const confirmPasswordInput = document.querySelector("#confirmPassword");
+  const signUpButton = document.querySelector("#signForm");
+  const togglePasswordIcon = document.querySelector(
+    "#togglePasswordVisibility"
+  );
+  const togglePasswordIcon2 = document.querySelector(
+    "#togglePasswordVisibility2"
+  );
+
+  // 이메일 입력 처리 함수 : 유효성 검사 및 에러 메시지 관리
+  function handleEmailInput() {
+    const emailValue = emailInput.value;
+    const emailError = document.querySelector("#emailError");
+
+    // 이메일 입력부분이 비어 있는 경우
+    if (!emailValue) {
+      emailError.textContent = "이메일을 입력해 주세요.";
+      emailInput.classList.add("invalid");
+      // 이메일 입력부분이 비어있지 않는 경우 + 이메일 형식이 유효하지 않은 경우
+    } else if (!isValidEmail(emailValue)) {
```

Comment:
코멘트는 연관 코드가 어떤 것인지 알 수 있게 작성해주시면 좋겠어요.

```suggestion
    } 
    // 이메일 입력부분이 비어있지 않는 경우 + 이메일 형식이 유효하지 않은 경우
    else if (!isValidEmail(emailValue)) {
```

Reply:
반영했습니다.